server/php/UploadHandler.php

```diff
@@ -1320,16 +1331,16 @@ public function generate_response($content, $print_response = true) {
             $redirect = stripslashes($this->get_post_param('redirect'));
             if ($redirect && preg_match($this->options['redirect_allow_target'], $redirect)) {
                 $this->header('Location: '.sprintf($redirect, rawurlencode($json)));
-                return;
+                return null;
```

Comment:
`return` should be the same as `return null`.
I would understand it in this context if you want to make it more explicit that the content returned is `null`.